rpc/bridge.go

```diff
@@ -209,6 +209,59 @@ func (b *BridgeEndpoints) InjectedInfoAfterIndex(networkID uint32, l1InfoTreeInd
 	)
 }
 
+// BridgesResult contains the bridges and the total count of bridges
+type BridgesResult struct {
+	Bridges []*bridgesync.Bridge `json:"bridges"`
+	Count   int                  `json:"count"`
+}
+
+func (b *BridgeEndpoints) GetBridges(
+	networkID uint32,
+	pageNumber, pageSize *uint32,
+	depositCount uint64,
```

Comment:
Should it be nullable, because AFAIK it should be an optional parameter

```suggestion
	depositCount *uint64,
```